monitoring/bin/deploy_monitoring_cluster.sh

```diff
@@ -229,6 +220,24 @@ else
   log_debug "PrometheusRule $MON_NS/v4m-kubernetes-apps does not exist"
 fi
 
+# Elasticsearch Datasource for Grafana
+# Moved down to make sure Grafana pods exist
+ELASTICSEARCH_DATASOURCE="${ELASTICSEARCH_DATASOURCE:-false}"
+if [ "$ELASTICSEARCH_DATASOURCE" == "true" ]; then
+  set +e
+  monitoring/bin/create_elasticsearch_datasource.sh
```

Comment:
I see no messages indicating successful deployment of the datasource.  Is that because the script itself will emit messages indicating success?

Reply:
There is a message in the create_elasticsearch_datasource.sh script that does it, but I'll add a DEBUG or INFO message in the deploy monitoring scripts to explicitly say that it's trying to create this data source.  It will be added in the next pull request.